OgcApi.SqlServer.Tests/SqlServerFacts.cs

```diff
@@ -578,5 +578,45 @@ public void DeleteNotExistingFeature()
             var provider = TestProviders.GetDefaultProvider();
             Assert.Throws<ArgumentException>(() => provider.DeleteFeature("PolygonsForInsert", "200"));
         }
+
+        [Fact]
+        public async void GetTile()
+        {
+            var tile = await TestProviders.GetDefaultProvider().GetTileAsync("Polygons", 1, 1, 1);
+            Assert.NotNull(tile);
+        }
+
+        [Fact]
+        public void GetTileUnknownCollection()
+        {
+            Assert.ThrowsAsync<ArgumentException>(() => TestProviders.GetDefaultProvider().GetTileAsync("test", 1, 1, 1));
+        }
+
+        [Fact]
+        public async void GetTileIncorrectZoomLevel()
+        {
+            var tile = await TestProviders.GetDefaultProvider().GetTileAsync("Polygons", 25, 162, 82);
+            Assert.Null(tile);
+        }
+
+        [Fact]
+        public async void GetTileIncorrectTileRow()
+        {
+            var tile = await TestProviders.GetDefaultProvider().GetTileAsync("Polygons", 8, 162, 90);
+            Assert.Null(tile);
+        }
+
+        [Fact]
+        public async void GetTileIncorrectTileCol()
+        {
+            var tile = await TestProviders.GetDefaultProvider().GetTileAsync("Polygons", 8, 170, 82);
+            Assert.Null(tile);
+        }
+
+        [Fact]
+        public void GetLimitsUnknownCollection()
+        {
+            Assert.Throws<ArgumentException>(() => TestProviders.GetDefaultProvider().GetLimits("test"));
```

Comment:
The test doesn't work. Logically, if you ask for TileMatrixLimits for a collection that doesn't exist in the provider's settings, you should generate an exception. See how this is done in MBTiles.